kim_cnn/test.py

```diff
@@ -0,0 +1,45 @@
+import torch

```

Comment:
Please replace all tabs with 4 spaces in this file.

Reply:
Ah sorry I missed this file when I was cleaning up. This was a file that I put all the test code in and shouldn't be uploaded. Thanks for spotting it out.